### PR TITLE
Order

### PR DESCRIPTION
**Pull request by salinda-yasas-99**

### CODE REVIEW

File: src/main/java/com/sms/businesslogic/controller/OrderController.java [hunk 1]
```diff
@@ -18,10 +18,16 @@ public class OrderController {
...
-    public List<Order> getAllOrders(){
+    public List<OrderDTO> getAllOrders(){
         return orderService.getAllOrders();
     }
 
+    @GetMapping("/orders/{userID}")
+    public List<OrderDTO> getAllOrdersByUserName(@PathVariable Integer userID){
+        return orderService.getAllOrdersByUserName(userID);
+    }
+
```
Comment (lakshankd) [inline src/main/java/com/sms/businesslogic/controller/OrderController.java:29]: remove unnecessary lines

File: src/main/java/com/sms/businesslogic/service/OrderService.java [hunk 2]
```diff
@@ -17,21 +19,66 @@
...
+
+        return OrderDTO.builder()
+                .orderId(order.getOrderId())
+                .totalQuantity(order.getTotalQuantity())
+                .totalPrice(order.getTotalPrice())
+                .orderDate(order.getOrderDate())
+                .orderStatus(order.getOrderStatus())
+                .userID(order.getUser().getUserId())
+                /*.paymentID(order.getPayment().getPaymentId())
+                .deliveryID(order.getDelivery().getDeliveryId())*/
```
Comment (lakshankd) [inline src/main/java/com/sms/businesslogic/service/OrderService.java:65]: remove this line

File: src/main/java/com/sms/businesslogic/controller/OrderController.java (hunk 1 above)
Comment (salinda-yasas-99) [reply src/main/java/com/sms/businesslogic/controller/OrderController.java:29]: solved

File: src/main/java/com/sms/businesslogic/service/OrderService.java (hunk 2 above)
Comment (salinda-yasas-99) [reply src/main/java/com/sms/businesslogic/service/OrderService.java:65]: solved

**Review by lakshankd — APPROVED** [review]

**Pull request merged**